go.mod

```diff
@@ -214,7 +214,6 @@ require (
 	github.com/cespare/xxhash/v2 v2.1.2 // indirect
 	github.com/chai2010/gettext-go v1.0.2 // indirect
 	github.com/coreos/go-systemd/v22 v22.3.3-0.20220203105225-a9a7ef127534 // indirect
-	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f // indirect
```

Comment:
@codingllama any idea what's going on with this one? It looks like all of the golang dep updates remove this line, but `go mod tidy` adds it back. To complicate matters, `go mod why github.com/coreos/pkg` reports that this module is not needed.

For now I've added a commit to this branch which just does `go mod tidy`, adding this line back

Reply:
I thought it was added too, but in any case we might as well do it here if it helps.

Reply:
Actually, I think the bot might tidy before merge, so let's try to fix it before submitting this PR. @nklaassen

Reply:
Okay this was a bit weird since `github.com/coreos/pkg` is not actually a package but just a module holding other packages. I eventually figured out the real dependency is `github.com/coreos/go-oidc/oidc` which imports `github.com/coreos/pkg/timeutil`.

`e/` does directly depend on `github.com/coreos/go-oidc/oidc` so I've added it to `e_imports.go` here. `go mod tidy` no longer tries to make any changes when `e/` is not present.

Reply:
Many thanks!